src/components/footer/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import logo from "./assets/ac_logo_24.png";
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    border: 0,
+    borderRadius: 3,
+    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+    height: "10vh",
+    padding: 0,
+  },
+
+}));
```

Comment:
I see what you mean and I think I found the reasoning. She is using `100vw` for the size of the component, this is fine if the page has no vertical scroll bar, but once it does, the `vw` doesn't include the width of the scroll bar and has to bump it out a bit. You can get around this by using `100%` of the body and it removes the horizontal scroll bar. @lalliere - change the width to be `100%` instead of `100vw` (Unless @danieltichiyama you know of a reason not to do that?)

See this SO thread - https://stackoverflow.com/questions/30489594/prevent-100vw-from-creating-horizontal-scroll